src/python/WMCore/MicroService/MSTransferor/DataStructs/Workflow.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, reqName, reqData, logger=None, verbose=False):
         self.pileupDatasets = set()
         self.pileupRSEList = set()
 
+        # flag to indicate if workflow is updated, e.g. see MSTransferor when we
+        # update site white/black lists and setup this flag
+        self.updatedWorkflow = False
```

Comment:
As I look further into the code, I think calling this variable like `self.dataReplacement` would be less ambiguous.

Reply:
Why?

How could naming a flag in a way that it literally reflects the purpose of a flag and the action it reflects to be less descriptive than a name which relates to its meaning through a background logic sitting on a second plan?

Reply:
ok, rename it.